hushline/__init__.py

```diff
@@ -104,4 +120,32 @@ def add_onion_location_header(response: Response) -> Response:
             )
             return response
 
+    # Register custom CLI commands
+    register_commands(app)
+
     return app
+
+
+def register_commands(app: Flask) -> None:
```

Comment:
You mean, define these commands in the premium blueprint instead of in the app?

That's a good idea, except now I have refactored things so that `flask stripe configure` runs each time in dev and prod and ensures the default tiers (which are required for non-premium Hush Line instances too) are created. And this PR is getting gnarly, so I think I will put this off for a follow-up PR.